src/wizard/github.jl

```diff
@@ -115,6 +115,15 @@ function obtain_token(; outs=stdout, github_api=GitHub.DEFAULT_API)
     end
 end
 
+function find_yggdrasil_commit_author_login(commit_hash; auth=github_auth(; allow_anonymous=true))
+  try
+    commit = GitHub.commit("JuliaPackaging/Yggdrasil", commit_hash; auth=auth)
+    commit.author.login
+  catch
+    nothing
+  end
+end
+
```

Comment:
I don't think this function belongs to this module, it isn't used anywhere here.  The function can also take the repository in input.  In the future we may want to make it easy to use something different from hard-coded `JuliaPackaging/Yggdrasil`.  Also, please use 4 spaces for indentation, like the rest of the package.

Reply:
I moved it close to `download_github_release()` in `src/AutoBuild.jl`